Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -601,6 +601,24 @@ static void updateSliderTrackPartForRenderer(SliderTrackPart& sliderTrackPart, c
     sliderTrackPart.setTickRatios(WTFMove(tickRatios));
 }
 
+static void updateSwitchThumbPartForRenderer(SwitchThumbPart& switchThumbPart, const RenderObject& renderer)
+{
+    ASSERT(is<HTMLInputElement>(renderer.node()->shadowHost()));
+    auto& input = downcast<HTMLInputElement>(*renderer.node()->shadowHost());
```

Comment:
I think this should be a `RELEASE_ASSERT`, since otherwise we have type confusion.